src/test/java/io/cryostat/net/web/http/api/v1/TargetRecordingOptionsPatchHandlerTest.java

```diff
@@ -159,6 +159,52 @@ public Map answer(InvocationOnMock args) throws Throwable {
         }
     }
 
+    @Test
+    void shouldUnsetRecordingOptions() throws Exception {
+        Map<String, String> defaultValues =
+                Map.of("toDisk", "true", "maxAge", "50", "maxSize", "32");
```

Comment:
Where do these default values come from (unless the values are arbitrary)?

Reply:
These are actually just arbitrary for this test. I should rename that variable.